GameBoard CD.puml

```diff
@@ -0,0 +1,68 @@
+@startuml
```

Comment:
A general comment on the diagram ... I don't see any links between players/groups, or groups/games etc (but I know those links exist).